apps/client/cloudbuild.yaml

```diff
@@ -58,7 +58,7 @@ steps:
           --region ${_REGION} \
           --service-account ${_SA} \
           --set-secrets MONGODB_URI=${_MONGODB_VAR}:latest \
-          --set-env-vars DISABLE_SEARCH_COUNTS=${_DISABLE_SEARCH_COUNTS} \
```

Comment:
I think this works but it's messy because the NEXT_PUBLIC env vars are supposed to be consumed at build time not at run time.  I think we should keep both actually since the new env var is a temporary hack.